src/TitleDetails/POLDetails/POLDetails.js

```diff
@@ -155,20 +155,20 @@ const POLDetails = ({
 
 POLDetails.propTypes = {
   accessProvider: PropTypes.string,
+  expectedReceiptDate: PropTypes.string,
   materialSupplier: PropTypes.string,
   orderFormat: PropTypes.string,
   orderType: PropTypes.string,
   poLineId: PropTypes.string,
   poLineNumber: PropTypes.string,
-  receiptDate: PropTypes.string,
   receivingNote: PropTypes.string,
   requester: PropTypes.string,
   rush: PropTypes.bool,
   vendor: PropTypes.string,
 };
 
 POLDetails.defaultProps = {
-  receiptDate: '',
+  expectedReceiptDate: '',
   receivingNote: '',
   rush: false,
```

Comment:
I is preferable to use default params.
See https://twitter.com/dan_abramov/status/1133878326358171650